server/chat-plugins/net-filters.ts

```diff
@@ -7,21 +7,22 @@
  */
 
 import {QueryProcessManager} from '../../lib/process-manager';
-import {FS} from '../../lib/fs';
-import {Utils} from '../../lib/utils';
+import {FS, Utils} from '../../lib/';
 import {Config} from '../config-loader';
 import {Repl} from '../../lib/repl';
+import {linkRegex} from '../chat-formatter';
 
 const PATH = "config/chat-plugins/net.json";
 const NUM_PROCESSES = Config.netfilterprocesses || 1;
 const PM_TIMEOUT = 2 * 60 * 60 * 1000; // training can be _really_ slow
 const WHITELIST = ["mia"];
 
 interface NetQuery {
-	data: string | TrainingLine[];
-	type: "run" | "train" | "save" | "load";
+	data: string | TrainingLine[] | AnyObject;
+	type: "run" | "train" | "save" | "load" | 'trainfrom';
 	options?: AnyObject;
 }
+type Model = import('brain.js').recurrent.LSTM;
```

Comment:
You can `ts-ignore` this if you want it to be `any` if `brain.js` isn't installed, I think?

I might also suggest calling it `LSTM` to match what brain.js calls it.

Reply:
Alright.